package.json

```diff
@@ -152,7 +152,8 @@
     "react-test-renderer": "^16.13.1",
     "request": "^2.88.2",
     "superdesk-code-style": "1.5.0",
-    "typescript-eslint-parser": "^18.0.0"
+    "typescript-eslint-parser": "^18.0.0",
+    "webdriver-manager": "12.1.7"
```

Comment:
Why are you adding "webdriver-manager"? I see it was removed not long ago in #3756.

Reply:
Never mind, I see you already added a comment in the PR regarding this.